src/components/node-list/node-list-tree.js

```diff
@@ -111,16 +115,25 @@ const TreeListProvider = ({
     }
   };
 
-  const renderModularPipelines = (treeData) =>
-    treeData.children.map((node) =>
+  const renderModularPipelines = (treeData, parentStatus) => {
+    // this value is needed to determine whether the children modular pipeline belongs to a parent under focusMode
+    const status =
+      parentStatus === false && treeData.id !== 'main'
```

Comment:
Is this `main` a convention on the frontend?

Reply:
yes, it is used to represent the top level representation of the tree ( as designated in the `node-list-items` selectors)